_posts/2022-04-26-hhvm-4.159.markdown

```diff
@@ -0,0 +1,21 @@
+---
+title: "HHVM 4.159"
+layout: post
+author: atry
+category: blog
+---
+
+HHVM 4.159 is released! This release marks the end of support of 4.153; 4.154 -
+4.158 remain supported, as does the 4.128 LTS release.
+
+# Highlights
+
+- Add an attribute `<<__SafeGlobalVariable>>` to mark global variables as safe,
```

Comment:
This attribute has been renamed at HEAD. Might be worth adding a note here for future readers. https://github.com/facebook/hhvm/commit/eea2b351e1db8370b496c0f0108e9e7898ee0764

Reply:
Thank you!